src/ElvClient.js

```diff
@@ -661,8 +681,39 @@ class ElvClient {
     };
   }
 
-  WriteTokenNodeUrl({writeToken}) {
-    const nodeUrl = this.HttpClient.draftURIs[writeToken];
+  /**
+   * Return node url for a given write token
+   *
+   * @methodGroup Nodes
+   * @namedParams
+   * @param {string} writeToken - The write token to match to a node
+   * @param {boolean=} networkCall - If specified, will make a network call to /nodes?token=<writeToken>
+   *
+   * @returns {Promise<string>} - The node url for a write token
+   */
+  async WriteTokenNodeUrl({writeToken, networkCall=false}) {
+    let nodeUrl;
+
+    if(networkCall) {
+      const nodes = await this.SpaceNodes({matchWriteToken: writeToken});
```

Comment:
I think the logic should be:

- get the nodeUrl from the cache (like before)
- if not found, then get it using the API

That way we don't need the networkCall parameter.